pipelines/h6c/rtp/v2/stage_1_task_scripts/_common.sh

```diff
@@ -113,3 +113,23 @@ function stringContain()
 {
   [ -z "${2##*$1*}" ] && [ -z "$1" -o -n "$2" ];
 }
+
+
+# >>> conda initialize >>>
+# contents within this block must be kept matching those in ~/.bashrc
+__conda_setup="$('/home/obs/mambaforge/bin/conda' 'shell.bash' 'hook' 2> /dev/null)"
+if [ $? -eq 0 ]; then
+    eval "$__conda_setup"
+else
+    if [ -f "/home/obs/mambaforge/etc/profile.d/conda.sh" ]; then
+        . "/home/obs/mambaforge/etc/profile.d/conda.sh"
+    else
+        export PATH="/home/obs/mambaforge/bin:$PATH"
+    fi
+fi
+unset __conda_setup
+
+if [ -f "/home/obs/mambaforge/etc/profile.d/mamba.sh" ]; then
+    . "/home/obs/mambaforge/etc/profile.d/mamba.sh"
+fi
+# <<< conda initialize <<<
```

Comment:
Is all this still necessary?

Reply:
It is.  For some reason conda/mamba are not activated in the inmost do_ scripts.  I tried out a few more elegant ways like sourcing .bashrc or making this all a bash function in _common. But there were problems with all. After 2 hours just on this one issue I had to put it aside to make forward progress.

Reply:
Ok, and are we confident this won't override the conda environment in other scripts (which is currently specified via the toml)? If not, perhaps we want to move this out of _common.sh and into the specific do_script?